src/Microsoft.ML.Transforms/CustomMappingTransformer.cs

```diff
@@ -203,12 +201,27 @@ public ITransformer GetTransformer()
     }
 
     /// <summary>
-    /// The <see cref="IEstimator{TTransformer}"/> to define a custom mapping of rows of an <see cref="IDataView"/>.
-    /// For usage details, please see <see cref="CustomMappingCatalog.CustomMapping"/>
+    /// Applies a custom mapping of input columns to output columns of an <see cref="IDataView"/>.
```

Comment:
>Applies a custom mapping of input columns to output columns of an <see cref="IDataView"/>. [](start = 8, length = 90)

Applies a custom mapping function to the specified input columns. The result will be in output columns.